src/pages/home/components/search/search.tsx

```diff
@@ -22,10 +22,16 @@ export const Search: FC<SearchProps> = ({ date, setDatePickerValue }) => {
 
   return (
     <StyledSection>
-      <Container>
+      <Container sx={{ display: "flex", flexDirection: "column" }}>
         <Typography
           id={ANCHORS.SEARCH}
-          sx={{ fontWeight: 400, textAlign: "center", color: "common.white" }}
+          sx={{
+            fontWeight: ({ theme: { fontWeights } }) => fontWeights.normal,
```

Comment:
в MUI теме можно стучаться к `fontWeight` аналогично как к цветам: 
```suggestion
            fontWeight: 'fontWeightRegular',
```

P.S. Ничего добавлять не нужно, в MUI уже прописано